source/development/contributing.rst

```diff
@@ -136,6 +139,34 @@ development directly on it. Follow these steps:
     * if the **reviewers** are taking too long, try contacting the :term:`PR`
       assignee.
 
+.. _split_prs:
+
+Splitting Large PRs
+###################
+
+Code reviews are typically a difficult and time-consuming process. To
+facilitate the reviewer's work, contributors should try to split their PRs into
+smaller ones if they feel that the PR introduces a large set of modifications.
+Nevertheless, the PR assignee decides if the PR is too large and may request
+splitting it.
+
+To split your large PR, a contributor should follow this process:
+
+* create a topic branch from the ``main`` branch and call it
+  ``wip/<feat_name>`` (where ``<feat_name>`` is a short description of the
+  feature, subsystem, or functionality being developed);
```

Comment:
I would add that external contributors contributors that cannot the wip branch directly should ask for the creation of this branch from a maintainer who will become responsible for managing the wip branch and synchronizing with the external contributor.

Reply:
Fixed.